tests/Controllers/AccountsControllerTests.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace staffinfo.divers.tests.Controllers
+{
+    public class AccountsControllerTests
+    {
```

Comment:
Why is this empty?

Reply:
Because Controller doesn't have any methods to check